cachito/workers/pkg_managers/gomod.py

```diff
@@ -813,7 +813,9 @@ def get_golang_version(module_name, git_path, commit_sha, update_tags=False, sub
     if update_tags:
         try:
             repo.remote().fetch(force=True, tags=True)
-        except:  # noqa E722
+        except Exception as ex:
```

Comment:
@chmeliik I was looking at this code block and wondering... shouldn't we fail the request if the `git fetch` fails to execute? The consequence here is that we would be potentially resolving a wrong version for the module, right? Or is there something I'm missing here?